### PR TITLE
refactoring for auto_onboarding

### PR DESCRIPTION
**Pull request by jinwon0325kim**
Refactoring was performed on the code related to auto onboarding.

### CODE REVIEW

File: src/auto_onboarding/auto_devicelayout.py [hunk 1]
```diff
@@ -49,52 +49,41 @@ def setupUi(self, parent, comport, vendor):
...
+
+        self.insert_comboBox()
+        QtCore.QMetaObject.connectSlotsByName(parent)
+
+    def init_items(self, comport, vendor):
         self.chkbox = QtWidgets.QCheckBox(self.layoutWidget)
-        self.chkbox.setObjectName("chkbox")
-        self.chkbox.setMinimumWidth(15)
-        self.chkbox.setMaximumWidth(30)
-        self.horizontalLayout.addWidget(self.chkbox)
```
Comment (junyong-sim) [inline src/auto_onboarding/auto_devicelayout.py:64]: How can we configure the attribute of these gui components after removing above codes ? 
Is it possible through .ui file ?

Comment (jinwon0325kim) [reply src/auto_onboarding/auto_devicelayout.py:64]: Yes, applied it to .ui file

File: src/auto_onboarding/auto_devicelayout.py [hunk 2]
```diff
@@ -49,52 +49,41 @@ def setupUi(self, parent, comport, vendor):
...
-        self.layoutWidget.setObjectName("widget")
         self.horizontalLayout = QtWidgets.QHBoxLayout(self.layoutWidget)
-        self.horizontalLayout.setContentsMargins(0, 0, 0, 0)
-        self.horizontalLayout.setObjectName("horizontalLayout")
+        self.horizontalLayout.setContentsMargins(8, 0, 0, 0)
+        self.init_items(comport, vendor)
+        for index, default_item in enumerate(parent.default_items):
+            self.add_item[index].setMinimumWidth(default_item.minimumWidth())
+            self.add_item[index].setMaximumWidth(default_item.maximumWidth())
+            self.horizontalLayout.addWidget(self.add_item[index])
```
Comment (gdinolee) [inline src/auto_onboarding/auto_devicelayout.py:63]: Repetitive code is compressed into one, improving management and readability.

**Review by gdinolee — APPROVED** [review]
Repetitive code is compressed into one, improving management and readability.

**Review by junyong-sim — APPROVED** [review]
LGTM

**Pull request merged**